examples/Test Output/Exporters/JSON/json_to_pdf.py

```diff
@@ -6,19 +6,20 @@
 import json
 
 from testplan.report import TestReport
-from testplan.exporters.testing import PDFExporter
-
 from testplan.report.testing.styles import Style, StyleEnum
 
+from testplan.exporters.testing import PDFExporter
+from testplan.exporters.testing.json import ATTACHMENT_1, ATTACHMENT_2
+
 
 def main(source, target):
 
     with open(source) as source_file:
         data = json.loads(source_file.read())
         if (
             data.get("version", 1) >= 2
```

Comment:
My personal preference would be merely check the version. (And we need to document v1/2 difference very clearly somewhere).